config.go

```diff
@@ -33,6 +33,14 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
```

Comment:
I'd like to move `Resources` as well. Use case is that of running exec processes in a sub-container inside the main container. Imagine not letting your editor exec session resulting in a OOM kill of the container.